<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplyTheMessage</class>
 <widget class="QDialog" name="ReplyTheMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="newlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>新回复：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="namelineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="replyTextEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>251</width>
     <height>160</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="contentTextEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>250</width>
     <height>160</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>联系人：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>内容：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replyButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>回复</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
